influunt-api/modules/influunt-core/app/models/Evento.java

```diff
@@ -190,9 +190,12 @@ public void setDataAtualizacao(DateTime dataAtualizacao) {
 
     @AssertTrue(groups = TabelaHorariosCheck.class, message = "Existem eventos configurados no mesmo dia e horário.")
     public boolean isEventosMesmoDiaEHora() {
-        if (!this.getTabelaHorario().getEventos().isEmpty() && this.getHorario() != null && this.getDiaDaSemana() != null && this.getTabelaHorario().getVersaoTabelaHoraria().getStatusVersao() == StatusVersao.EDITANDO) {
+        if (!this.getTabelaHorario().getEventos().isEmpty() &&
+            this.getHorario() != null && this.getDiaDaSemana() != null &&
+            this.getTabelaHorario().getVersaoTabelaHoraria().getStatusVersao() == StatusVersao.EDITANDO &&
```

Comment:
`this.getTabelaHorario().getVersaoTabelaHoraria().getStatusVersao() == StatusVersao.EDITANDO` funciona da mesma forma que `StatusVersao.EDITANDO.equals(this.getTabelaHorario().getVersaoTabelaHoraria().getStatusVersao())` neste caso?